pyproject.toml

```diff
@@ -10,8 +10,8 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-pandas = "^1.0.4"
+python = "^3.7.1"
```

Comment:
I updated these dependencies before I saw this PR. Please grab the pyproject.toml and poetry.lock files off of master.